tests/mpi/hp_step-40_variable_01.with_petsc=true.mpirun=10.output

```diff
@@ -0,0 +1,17 @@
+
+Cycle 0:
+   Number of active cells:       1024
+      104+100+104+100+104+104+100+104+100+104+
+   Number of degrees of freedom: 4225
+      457+412+433+416+417+429+417+418+404+422+
+DEAL:mpi::Solver stopped within 10 - 10 iterations
+   Solved in 10 iterations.
+
+Cycle 1:
+   Number of active cells:       4096
+      408+412+408+412+408+408+412+408+412+408+
+   Number of degrees of freedom: 16649
```

Comment:
Any idea why the number of degrees of freedom is different? The mesh is uniform and the number of cells is the same for all the test cases so I don't understand why the number of dofs is different.

Reply:
Because you can't divide 4096 cells equally between 10 processors? ;-)

Reply:
You can't divide 4096 cells equally between 3 processors either yet for 1, 3, and 4 processors you have 4096 cells and 16641 degrees of freedom. Only for 10 processors, you have 4096 cells and 16649 degrees of freedom.

Reply:
Oh, now I see what you mean. I thought you were looking at line 12 and were suggesting that all processors should have the same number of cells.

This points to a bug. Would you be willing to merge this test for now (since it tests *something* useful by itself), and @marcfehling and I will investigate when he's back in 2 weeks.

Reply:
Should we change the output so that with 10 processors we also have 16641 dofs?

Reply:
> Should we change the output so that with 10 processors we also have 16641 dofs?

I agree that the blessed output should reflect the correct result.

Reply:
> There is a bug in the code that we will need to find. Upon fixing it, we will also change the output of this test to 16641 to ensure that the test continues to work.

Would you mind please opening up an issue for this so that its not forgotten?

Reply:
See #7162 now.

Reply:
Thank you!